doc/getting_started.md

```diff
@@ -457,15 +459,12 @@ the specified file paths.
 
 ## Capturing Power Traces
 
-There are currently two different scripts available for capturing OpenTitan
-power traces:
+There is a one single script available for capturing OpenTitan power traces:
 
-* `simple_capture_traces.py`: Supports capturing AES and SHA3 (KMAC) power traces
-  using either a CW-Husky or CW-Lite capture board.
-
-* `simple_capture_traces_batch.py`: Supports capturing AES power traces
-  traces in batches to achieve substantially higher capture rates (ca.
-  1300 traces/s). Requires the CW-Husky capture board.
+* `capture.py`: Supports capturing AES and SHA3 (KMAC) power traces
+  using either a CW-Husky or CW-Lite capture board. It supports also capturing 
+  AES power traces traces in batches to achieve substantially higher capture 
+  rates (ca. 1300 traces/s). Requires the CW-Husky capture board.
```

Comment:
`When using a CW-Husky board, the script also supports capturing AES power traces in batches to achieve substantially higher capture rates (ca. 1300 traces/s).`